frontend-react/src/pages/submissions/SubmissionTable.test.tsx

```diff
@@ -76,59 +80,4 @@ describe("SubmissionTable", () => {
         const rows = within(tBody).getAllByRole("row");
         expect(rows).toHaveLength(2);
     });
-
-    describe("when the numbered pagination feature flag is on", () => {
```

Comment:
The diff doesn't show it because this is a nearly identical test to the prior one, but this test _does_ remain (sans mockCheckFlag) to test numbered pagination on SubmissionTable